InventoryManagement/src/main/java/com/ccs/inventorymanagement/domain/Clothing.java

```diff
@@ -0,0 +1,64 @@
+package com.ccs.inventorymanagement.domain;
+
+import com.ccs.inventorymanagement.repo.ClothingEntity;
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder(toBuilder = true)
+public class Clothing extends Item {
+    public enum Apparel_type {
```

Comment:
Underscores should only be used in "static final" fields which are all capitalized.

I would just name this "Type" since it is within the Clothing class already, you would reference it like "Clothing.Type"

Reply:
Updated Apparel_type -> Type